bazel/utils/files.bzl

```diff
@@ -0,0 +1,90 @@
+load("@bazel_skylib//lib:paths.bzl", "paths")
+
+""" Changes a path at a specified keyword to another path, appending the remaining onto it
+Examples:
+  >>> trim_path("/tree/foo/bar", "foo", "/etc")
+  /etc/bar
+"""
```

Comment:
this docstring is documenting a trim_path, that does not exist in this file? Did you intend to rename `rebase_path`?

If this is the docstring for rebase_path, it should be tied to the `rebase_path` function, not at file level.